openassessment/templates/openassessmentblock/response/oa_response.html

```diff
@@ -50,7 +50,7 @@ <h4 class="step__title">
                     {% if submission_due %}
                     {% trans "You can save your progress and return to complete your response at any time before the due date" %}
                     (<span class="step__deadline">
-                        <span class="date ora-datetime" data-datetime="{{ submission_due|timezone:'UTC'|date:'c' }}" data-timezone="{{ user_timezone }}" data-format="longDateTime" data-language="{{ user_language }}"></span>
+                        <span class="date ora-datetime" data-datetime="{{ submission_due|timezone:'UTC'|date:'d/M/y' }}" data-timezone="{{ user_timezone }}" data-format="longDateTime" data-language="{{ user_language }}"></span>
                     </span>).
```

Comment:
@diegomillan For some reason the tag `data-datetime` does not takes effect for the format that is passed. Any clue?